### PR TITLE
ft(health): add Too many requests error

### PR DESCRIPTION
**Pull request by ThibaultRiviere**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "ioredis": "2.4.0",
     "ipaddr.js": "1.2.0",
     "joi": "^10.6",
-    "level": "~1.6.0",
+    "level": "scality/level#proto/custom-env",
```
Comment (DavidPineauScality) [inline package.json]: dropme ?

Comment (ThibaultRiviere) [reply package.json]: Yes it's because of the conflict between the deps of arsenal and MD . Will remove that when everything is finished !

Comment (DavidPineauScality) [reply package.json]: Ok, but could we not already merge this PR ? It does not break anything, and at least we'll be one PR down :)

Comment (ThibaultRiviere) [reply package.json]: Ok :)

**Comment by ThibaultRiviere** [comment]
@DavidPineauScality can be merge

**Comment by DavidPineauScality** [comment]
Merging, as this is non-colliding with any other changes in the project; and allow related PRs to make progress.
@ironman-machine r+

**Pull request merged**